cloud-movies-be/lib/api-gw/api-gw-stack.ts

```diff
@@ -135,5 +159,13 @@ export class ApiGwStack extends cdk.Stack {
         validateRequestParameters: true,
       },
     });
+
+    const searchMoviesLambdaIntegration = new apigateway.LambdaIntegration(searchMoviesFn);
+    const searchMoviesResource = api.root.addResource("search-movies");
+    searchMoviesResource.addMethod("POST", searchMoviesLambdaIntegration, {
+      requestValidatorOptions: {
+        validateRequestBody: true,
+      },
```

Comment:
If you want to validate the request body, you have to provide the body schema as in uploadMovieFn integration.